ext/intl/uchar/uchar.c

```diff
@@ -357,9 +361,12 @@ IC_METHOD(getPropertyValueName) {
 	zend_long property, value, nameChoice = U_LONG_PROPERTY_NAME;
 	const char *ret;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ll|l", &property, &value, &nameChoice) == FAILURE) {
-		RETURN_THROWS();
-	}
+	ZEND_PARSE_PARAMETERS_START(2, 3)
+		Z_PARAM_LONG(property)
+		Z_PARAM_LONG(value)
+		Z_PARAM_OPTONAL
```

Comment:
```suggestion
		Z_PARAM_OPTIONAL
```

Reply:
oh I passed the wrong configure option locally :)